samza-test/src/main/java/org/apache/samza/test/framework/TestRunner.java

```diff
@@ -186,7 +186,7 @@ public TestRunner addConfigs(Config config) {
   public TestRunner addOverrideConfig(String key, String value) {
     Preconditions.checkNotNull(key);
     Preconditions.checkNotNull(value);
-    configs.put(key, value);
+    configs.put(String.format("jobs.%s.%s", JOB_NAME, key), value);
```

Comment:
Rather, I would recommend to move the JobNode.CONFIG_JOB_PREFIX to JobConfig and use it here, instead of hard code the string.

Reply:
Done.